CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ project(collab-vm-server)
 include(cmake/CollabVM.cmake)
 
 set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++20")
```

Comment:
this is already guaranteed by CMake's C++ standard variable, no?

Reply:
On gcc 10.2 with "Unix Makefiles" as the cmake type it does not use the standards variable. It might be different for ninja or clang.